AnnoyancesFilter/Other/sections/self-promo.txt

```diff
@@ -137,6 +137,8 @@ www.kinopoisk.ru##div[class^="trailer-promo"]
 ! END: Yandex
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/176525
+adobe.com##.promobarModal
```

Comment:
to popups maybe?

Reply:
even if it's actually a self-promo?

Reply:
as well as a popup

Reply:
I think it's ok to move it to popups.

Reply:
so 'popup' prevails over 'self-promo' in this case and the rule should be moved to 
`AnnoyancesFilter/Popups/sections/popups_specific.txt` ?

Reply:
Yes.